android/intltestapp/src/main/res/values/strings.xml

```diff
@@ -0,0 +1 @@
+<resources></resources>
```

Comment:
Is this useful?

Reply:
It should be deletable.. Will do when touching the source next time.